src/configs/starship_root.rs

```diff
@@ -87,3 +85,27 @@ impl<'a> Default for StarshipRootConfig<'a> {
         }
     }
 }
+
+impl<'a> ModuleConfig<'a> for StarshipRootConfig<'a> {
+    fn load_config(&mut self, config: &'a toml::Value) {
+        if let toml::Value::Table(config) = config {
+            config.iter().for_each(|(k, v)| match k.as_str() {
+                "format" => self.format.load_config(v),
+                "scan_timeout" => self.scan_timeout.load_config(v),
+                "command_timeout" => self.command_timeout.load_config(v),
+                "add_newline" => self.add_newline.load_config(v),
+                unknown => {
+                    if !ALL_MODULES.contains(&unknown) && unknown != "custom" {
+                        log::warn!("Unknown config key '{}'", unknown);
```

Comment:
Would this log both config key and module name or just the key?

Reply:
This only logs the config key. But the logger adds `(starship::configs::module_name)` to the message.

Reply:
Thought so, wanted to confirm though. 👍